### DISCUSSION TITLE
[Feature] Application for laptop (MacOS, Linux or Windows)

### DISCUSSION THREAD

**Discussion by marcusjus** [immich-app/immich#11817] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

It would be nice to have an app for laptop (MacOS, Linux and Windows), like digikam, Photos...

### Platform

- [ ] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#1427
